contentcuration/contentcuration/celery.py

```diff
@@ -12,7 +12,7 @@
 
 # Using a string here means the worker will not have to
 # pickle the object when using Windows.
-app.config_from_object('django.conf:settings')
+app.config_from_object('django.conf:settings', namespace='CELERY')
```

Comment:
Note: Celery 4 changed the settings definition format. I had to do this to get the CELERY_TASK_ALWAYS_EAGER setting picked up. It's possible some of our existing settings weren't being detected after we upgraded to 4, so we may have been unintentionally running with default values.

Reply:
Oooh that's a good pointer! We'll find out I guess when we deploy to develop!